force-app/main/default/classes/MyControllerTest.cls

```diff
@@ -0,0 +1,8 @@
+@isTest(seeAllData=true)
+public with sharing class MyControllerTest {
+    
+    @isTest
+    public static void test(){
```

Comment:
PMD Violation: 
Apex unit tests should System.assert() or assertEquals() or assertNotEquals()